config/flow_template.j2

```diff
@@ -0,0 +1,203 @@
+# cycl template mytemplate.j2
+[scheduler]
+    allow implicit tasks = True
+[task parameters]
+    # Update this variable to match the range of parameter sets
+    param_set = 0..1
```

Comment:
Make this auto update to the parameter set length

Reply:
Good catch!

Reply:
Updated this to auto populate based on the number of rows passed in from CSV.